### PR TITLE
Add download links for optaweb-vehicle-routing distribution zip

### PR DESCRIPTION
**Pull request by cuijulian**
Jira: https://issues.redhat.com/browse/PLANNER-1778

### CODE REVIEW

File: download/download.html.haml [hunk 1]
```diff
@@ -206,3 +219,9 @@ title: Download
   %div(id="optaweb-employee-rostering-older" class="tab-pane fade")
     :asciidoc
       For older community releases, check https://download.jboss.org/drools/release/[the release archive].
+  %div(id="optaweb-employee-rostering-older" class="tab-pane fade")
```
Comment (yurloc) [inline download/download.html.haml]: It seems this duplicates the div above.

File: download/download.html.haml [hunk 2]
```diff
@@ -206,3 +219,9 @@ title: Download
...
+    :asciidoc
+      For older community releases, check https://download.jboss.org/drools/release/[the release archive].
+  %div(id="optaweb-vehicle-routing-older" class="tab-pane fade")
+    :asciidoc
+      For older community releases, check https://download.jboss.org/drools/release/[the release archive].
```
Comment (yurloc) [inline download/download.html.haml]: `s/drools/optaplanner/`

File: download/download.html.haml [hunk 3]
```diff
@@ -195,14 +206,19 @@ title: Download
...
+    %a(href="#optaweb-vehicle-routing-older" data-toggle="pill") OptaWeb Vehicle Routing
 
 %div(class="tab-content download-tab-content")
   %div(id="engine-older" class="tab-pane fade in active")
     :asciidoc
       For older community releases, check https://download.jboss.org/optaplanner/release/[the release archive].
   %div(id="execution-server-older" class="tab-pane fade")
     :asciidoc
-      For older community releases, check https://download.jboss.org/drools/release/[the release archive].
+      For older community releases, check https://download.jboss.org/optaplanner/release/[the release archive].
```
Comment (yurloc) [inline download/download.html.haml]: No, execution server distribution is actually available under `/drools`.

**Comment by ge0ffrey** [comment]
@cuijulian I merged this already to fix the broken pdf link live. It should be published by the time when you're online (go to optaplanner.org -> website info (footer on bottom) to see the actual "rake build" time that is live). Can you check if all Jiri's comments are resolved and if there are no broken links?

**Comment by cuijulian** [comment]
@ge0ffrey Thanks for merging, all links are working :+1: Looks like the build time was just an hour before I logged on.

**Pull request merged**